Open Judge System/Web/OJS.Web/Areas/Administration/Controllers/ProblemsController.cs

```diff
@@ -176,11 +184,20 @@ public ActionResult Create(int id, HttpPostedFileBase testArchive, DetailedProbl
                 this.AddResourcesToProblem(newProblem, problem.Resources);
             }
 
-            if (testArchive != null && testArchive.ContentLength != 0)
+            if (problem.AdditionalFiles != null && problem.AdditionalFiles.ContentLength != 0)
+            {
+                using (var archiveStream = new MemoryStream())
```

Comment:
Can we make extension of `HttpPostedFileBase` for this? The result will be `newProblem.AdditionalFiles = problem.AdditionalFiles.ToByteArray()`